README.md

```diff
@@ -194,13 +194,19 @@ const App = ({ message }: AppProps): JSX.Element => <div>{message}</div>;
 
 // you can also inline the type declaration; eliminates naming the prop types, but looks repetitive
 const App = ({ message }: { message: string }) => <div>{message}</div>;
+
+// Alternatively, you can use `React.FunctionComponent` (or `React.FC`), if you prefer.
```

Comment:
I think it would be good to denote that `React.FunctionComponent` and `React.FC` are effectively the same, to not give the impression that you have two different types to choose from. 

```suggestion
// Alternatively, you can use `React.FunctionComponent` (or the shorthand `React.FC`), if you prefer.
```